### PR TITLE
Update VI locale for v2

### LINKED ISSUE #828: VI locale migration
Hi team, 

I noticed that PR #700 disabled many locales including VI in which from my understanding that there are some changes to the API for the final v2. 

My question is how the old locales are going to be migrated? Do you need locale authors to help with that? If yes, do you have documentation related to the locale migration?

### PR DESCRIPTION
**Pull request by lihop**
Closes #828

### CODE REVIEW

File: src/locale/vi/_lib/localize/index.js [hunk 1]
```diff
@@ -1,53 +1,118 @@
 import buildLocalizeFn from '../../../_lib/buildLocalizeFn/index.js'
-import buildLocalizeArrayFn from '../../../_lib/buildLocalizeArrayFn/index.js'
 
-// Vietnammese locale reference: http://www.localeplanet.com/icu/vi-VN/index.html
-// Capitalization reference: http://hcmup.edu.vn/index.php?option=com_content&view=article&id=4106%3Avit-hoa-trong-vn-bn-hanh-chinh&catid=2345%3Atham-kho&Itemid=4103&lang=vi&site=134
-var weekdayValues = {
```
Comment (kossnocorp) [inline src/locale/vi/_lib/localize/index.js]: @lihop aren't these references are still actual and worth keeping?

Comment (lihop) [reply src/locale/vi/_lib/localize/index.js]: Yes. I will add them back in.

**Comment by lihop** [comment]
I have removed vi from the outdated languages list and re-added the locale references.

**Comment by kossnocorp** [comment]
Kudos to you, guys! I'm going to merge few more pending locales and publish a release. Stay tuned!

**Comment by kossnocorp** [comment]
I've just published a new version with this locale `date-fns@2.0.0-alpha.18`. See the change log: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha18

Thank you again for your work!

**Pull request merged**